src/Symfony/Component/DependencyInjection/Loader/FileLoader.php

```diff
@@ -217,6 +227,41 @@ public function registerAliasesForSinglyImplementedInterfaces(): void
         $this->interfaces = $this->singlyImplemented = $this->aliases = [];
     }
 
+    final protected function loadExtensionConfig(string $namespace, array $config): void
+    {
+        if (!$this->prepend) {
+            $this->container->loadFromExtension($namespace, $config);
+
+            return;
+        }
+
+        if ($this->importing) {
+            if (!isset($this->extensionConfigs[$namespace])) {
+                $this->extensionConfigs[$namespace] = [];
+            }
+            array_unshift($this->extensionConfigs[$namespace], $config);
```

Comment:
```suggestion
            array_unshift($this->extensionConfigs[$namespace] ??= [], $config);
```

Reply:
This one cannot be applied because https://3v4l.org/f6t7FP